packages/platform/platform-http/src/testing/FakeResponse.ts

```
@@ -25,6 +39,35 @@ export class FakeResponse extends EventEmitter {
     this.set("content-length", content);
   }
 
+  cookie(name: string, value: string, options?: TsED.SetCookieOpts) {
+    const opts = {...options};
+
+    const val = typeof value === "object" ? "j:" + JSON.stringify(value) : String(value);
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)